docs/stlats.md

```diff
@@ -196,6 +196,8 @@ function generateSoulscream(player) {
 
 Because JavaScript works with double-precision floating point numbers, it's unable to represent numbers over 10^309. With a `soul` value of larger than 309, the `magnitude` variable above will become exactly zero (`1 / Infinity`), a division by zero will occur in the `stlatDigit` calculation, and the resulting letter will instead be the word `undefined`. This explains why [Chorby Soul](https://www.blaseball.com/player/a1628d97-16ca-4a75-b8df-569bae02bef9) has 3400 valid soulscream letters (309 * 11 = 3399) until it just becomes the word `undefined` repeated over and over :)
 
+After Chorby Soul was reintroduced to the league due to necromancy after Season 14, soulscreams now only include letters for the first 300 soul. Players with soul larger than 300 will end with "... (CONT. FOR X SOUL)", where X is `soul - 300`.
+
 ## Blood type
 Player blood type is usually cosmetic, but in the case of team-wide blood type blessings, may have an effect.
```

Comment:
The information about blood type being required for blood modifiers is no longer true, right?

Reply:
I believe so, yes. Certainly the part about it changing blood type (It didn't for Crabs or Steaks, but did for Tigers and Talkers).

Reply:
IIRC Charm was the last one to require the actual blood type, but I'm not sure of the exact timeline of when they changed them all. If we want to be certain we can go back and check players like Baldwin Breadwinner in S14, other Lovers players in recent seasons, etc...

Also the type didn't change for Crabs and Talkers, but the feed events were there for their blood type changes, so I guess the change part itself was just bugged for one election.